heroic-core/src/main/java/com/spotify/heroic/metric/LocalMetricManager.java

```diff
@@ -372,6 +374,9 @@ public QueryTrace buildTrace() {
                 dataInMemoryReporter
             );
 
+            // Add watcher to set to compute stats across all queries
+            quotaWatchers.add(quotaWatcher);
```

Comment:
I'm having trouble understanding why this line wasn't there in the first place. Do we see any Quota-y stuff in the logs?

Reply:
I've just introduced this hashset specifically to get global numbers